src/server/middlewares/handleRender.js

```diff
@@ -71,7 +71,7 @@ const Html = ({ content, state }) => (
     <body>
       <div id='root' dangerouslySetInnerHTML={{ __html: content }} />
       <script dangerouslySetInnerHTML={{
-        __html: `window.__APOLLO_STATE__=${JSON.stringify(state)};`
+        __html: `window.__APOLLO_STATE__=${JSON.stringify(state)};window.__CONFIG__=${JSON.stringify(config.get('server'))}`
```

Comment:
There are not environment (ex, process.env.NODE_ENV) variable on the client side so I have to pass variables to the browser by rendering them to windows variable of a browser.

Anyway, I found that `createNetworkInterface` will call to same origin domain that is requested if `uri` is a string without a domain name. Then, I will remove `window.__CONFIG__` following reason,
1.client side will call to server side with the same hostname that client side was called.
2.You(@pNagi) already proxy `api server` to `ssr server`.
3.In **Apollo** project, It alway calls to the same original domain.

Please check the code below,
```javascript
const networkInterface = createNetworkInterface({
  uri: `/graphql`, // uri with out domain name
  opts: {
    credentials: 'same-origin'
  },
})
```